### PR TITLE
Bug/1432/disable map highlighting on drag or move

### PR DESCRIPTION
**Pull request by RomanenkoVladimir**
# Disable map highlighting on drag or move

_**Please read the [CONTRIBUTING.md](https://github.com/MaibornWolff/codecharta/blob/main/CONTRIBUTING.md) before opening a PR.**_

closes: #1432

## Description

**Descriptive pull request text**, answering:
- disables highlighting when cursor is in dragging or moving mode
- does not fire highlight events in those modes reducing overhead

### CODE REVIEW

File: visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts [hunk 1]
```diff
@@ -282,21 +282,20 @@ describe("codeMapMouseEventService", () => {
...
 			codeMapMouseEventService.updateHovering()
 
-			expect(threeSceneService.addBuildingToHighlightingList).toHaveBeenCalledWith(CODE_MAP_BUILDING)
+			expect(threeSceneService.addBuildingToHighlightingList).toHaveBeenCalled()
 			expect(threeSceneService.highlightBuildings).toHaveBeenCalled()
 			expect(document.body.style.cursor).toEqual(CursorType.Pointer)
 		})
 
-		it("should hover a node when no node is hovered, and an intersection was found but not change cursor in grabbing mode", () => {
+		it("should not hover node when an intersection was found and the cursor is set to grabbing", () => {
```
Comment (ce-bo) [inline visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts:298]: Thanks for shortening the test names to make them more readable. I think we should replace the word "hover" with "highlight" That applies to the other test as well. What do you think?

**Review by ce-bo — APPROVED** [review]
LGTM. However, I have suggested a small improvement.

File: visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts (hunk 1 above)
Comment (RomanenkoVladimir) [reply visualization/app/codeCharta/ui/codeMap/codeMap.mouseEvent.service.spec.ts:298]: Sounds good, I will add that

File: visualization/package.json [hunk 2]
```diff
@@ -127,6 +127,7 @@
 		"@types/puppeteer": "^3.0.2",
 		"@typescript-eslint/eslint-plugin": "^4.0.1",
 		"@typescript-eslint/parser": "^4.0.1",
+		"@webpack-cli/serve": "^1.0.1",
```
[the remark this answers is not in the document]
Comment (RomanenkoVladimir) [reply visualization/package.json:130]: No I was confused about that bit as well, I got a warning that npm i was moved and i need webpack.cli/serve
Should I remove the change?

[the remark this answers is not in the document]
Comment (RomanenkoVladimir) [reply visualization/package.json:130]: I investigated a bit and it seems it was an issue with my project setup
It should not be required anymore
Thanks for the comment on the matter :)

**Review by BridgeAR — APPROVED** [review]

**Pull request merged**